src/components/tx-flow/flows/TokenTransfer/CreateTokenTransfer.tsx

```diff
@@ -56,7 +55,7 @@ const CreateTokenTransfer = ({
   const safeTokenAddress = getSafeTokenAddress(chainId)
   const isSafeTokenPaused = useIsSafeTokenPaused()
   const isOnlySpendingLimitBeneficiary = useIsOnlySpendingLimitBeneficiary()
```

Comment:
I think we need to `allow` this for non-owners beneficiaries.

Reply:
It is allowed.

Reply:
`allow` is `false` [by default](https://github.com/safe-global/safe-wallet-web/blob/refactor-spending/src/hooks/useIsOnlySpendingLimitBeneficiary.tsx#L5).

Reply:
Ah, gotcha, thanks!